RocketChat/Rocket.Chat
# fix: Quote attachments are not deleted on message update

## Proposed changes (including videos or screenshots)

 - Remove old quote attachments from messages on every update.

## Issue(s)


## Steps to test or reproduce

There are two main test cases:
1. Update a message to remove all of its quotes: no quote attachment should be displayed within the message after the update;
2. Update a message to remove some of its quotes: only quote attachments related to links that are still in the message should be displayed.

This can be tested through the UI **OR** using the `updateMessage` method or `chat.update` endpoint.

## Further comments

SUP-411
